src/main/java/com/example/sweets/Sweet.java

```diff
@@ -0,0 +1,60 @@
+package com.example.sweets;
+
+public class Sweet {
+    private int id;
+    private String name;
+    private String company;
+    private int price;
+    private String prefecture;
+
+    public Sweet(int id, String name, String company, int price, String prefecture) {
+        this.id = id;
+        this.name = name;
+        this.company = company;
+        this.price = price;
+        this.prefecture = prefecture;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getCompany() {
+        return company;
+    }
+
+    public int getPrice() { return price;
+    }
+
+    public String getPrefecture() {
+        return prefecture;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setCompany(String company) {
+        this.company = company;
+    }
+
+    public void setPrice(int price) throws IllegalArgumentException {
```

Comment:
IllegalArgumentException は実行時例外なのでthrowsなくても大丈夫かなと思います。
```suggestion
    public void setPrice(int price) {
```

Reply:
対応しました！
src/main/java/com/example/sweets/Sweet.java
0348e5467a2c58a08736032f695bfb2fb5275a22